README.md

```diff
@@ -82,10 +82,24 @@ Execute the commands below to build from source.
 6. To debug the tests:
 
         ./gradlew clean build -Pdebug=<port>
+        ./gradlew clean test -Pdebug=<port>
 
 7. To debug the package with Ballerina language:
 
-        ./gradlew clean build -PbalJavaDebug=<port>     
+        ./gradlew clean build -PbalJavaDebug=<port>
+        ./gradlew clean test -PbalJavaDebug=<port>
+
+8. Publish ZIP artifact to the local `.m2` repository:
+   
+        ./gradlew clean build publishToMavenLocal
+   
+9. Publish the generated artifacts to the local Ballerina central repository:
+   
+        ./gradlew clean build -PpublishToLocalCentral=true
```

Comment:
Github actions also needs to be updated

Reply:
checked the publish-release.yml file and no changes needed in that file.